app/views/admin/users/show.html.erb

```diff
@@ -1,10 +1,16 @@
 <div class="grid gap-6">
   <div class="crayons-card p-6">
     <h2 class="d-inline">
-      <%= @user.name %><%= link_to "@#{@user.username}", @user.path, class: "ml-2", target: "_blank", rel: "noopener" %>
+      <% if @user.registered? %>
+        <%= @user.name %><%= link_to "@#{@user.username}", @user.path, class: "ml-2", target: "_blank", rel: "noopener" %>
+      <% else %>
+        @<%= @user.username %> is not registered.
```

Comment:
**(thought, non-blocking)** : I had to read through the write up to clarify what a "non-registered" user is, hence I feel like when an admin see's this message ` @<%= @user.username %> is not registered.`, they also will probably not know what it means. 

Their thought process is probably going to be 'what does a non-registered user mean?', can I get them `registered`. They may not be sure whether the action to get the user registered lies with them or the user. 

So I was thinking that perhaps we can re-phrase this in a more user friendly way for the admins. 

Possibilities that come to mind include:
` @<%= @user.username %> has not accepted their invite as yet` (not sure if this is actually accurate) or  ` @<%= @user.username %> has not gone through gone through the onboarding process` or something more accurate to provide the admin with context on what _not registered_ means.

Reply:
Agreed! I thought about potentially making the message something along the lines of "`@<%= @user.username %> has not accepted their invite as yet`," but because I still am having trouble verifying whether or a not a user becomes "registered" after accepting the invite and completing the onboarding process, I left it as is. If, upon acceptance and onboarding completion they do move from "`registered: false`" to "`registered: true`" then I will absolutely make the change to your suggested copy, @Ridhwana! 😊

Reply:
Perfect, this sounds like a good plan 🎉

Reply:
I've made this change now that I was able to confirm that a user's column goes from "`registered: false`" to "`registered: true`" upon acceptance of an invitation! cc: @Ridhwana 😄